nano/lib/config.hpp

```diff
@@ -3,21 +3,24 @@
 #include <chrono>
 #include <cstddef>
 
-namespace rai
+namespace nano
 {
 /**
  * Network variants with different genesis blocks and network parameters
  * @warning Enum values are used for comparison; do not change.
  */
-enum class rai_networks
+enum class nano_networks
 {
 	// Low work parameters, publicly known genesis key, test IP ports
+	nano_test_network = 0,
 	rai_test_network = 0,
```

Comment:
Is keeping both the rai and nano enum values intentional?

Reply:
I was leaving it in for backwards compatibility, but the option name that is specified was also renamed with no backwards compatibility so maybe it's not needed -- or maybe more backwards compatibility is needed.